src/AutoRest.CSharp/Mgmt/Generation/ResourceContainerWriter.cs

```diff
@@ -226,6 +228,21 @@ private void WriteContainerMethodScope(bool isAsync, FormattableString returnTyp
             }, catch404);
         }
 
+        private void WriteGetFromRestClient(RestClientMethod method, bool async)
+        {
+            var parameterMapping = BuildParameterMapping(method);
+            string await = async ? "await " : string.Empty;
+            string asyncSuffix = async ? "Async" : string.Empty;
+            string configAwait = async ? ".ConfigureAwait(false)" : string.Empty;
+            _writer.Append($"var response = {await}{RestClientField}.{method.Name}{asyncSuffix}(");
+            foreach (var parameter in parameterMapping)
+            {
+                _writer.AppendRaw(parameter.IsPassThru ? parameter.Parameter.Name : parameter.ValueExpression);
+                _writer.AppendRaw(", ");
+            }
+            _writer.Line($"cancellationToken: cancellationToken){configAwait};");
+        }
+
         private void WriteGetVariants(RestClientMethod method)
         {
             var parameterMapping = BuildParameterMapping(method);
```

Comment:
Remove this line as parameterMapping is no longer used here.

Reply:
its used for the passthrough params used in the write method scope